MyConfig.h

```diff
@@ -249,6 +249,102 @@
 //#define MY_RS485_HWSERIAL (Serial1)
 /** @}*/ // End of RS485SettingGrpPub group
 
+
+
+
+
+/**
+ * @defgroup HC12SettingGrpPub HC12
+ * @ingroup RadioSettingGrpPub
+ * @brief These options are specific to the HC12 serial radio transport.
+ * @{
+ */
+
+/**
+ * @def MY_HC12
+ * @brief Define this to use the HC12 wired transport for sensor network communication.
+ */
+//#define MY_HC12
+
+/**
+ * @def MY_HC12_BAUD_RATE
+ * @brief The HC12 BAUD rate.
+ */
+#ifndef MY_HC12_BAUD_RATE
+#define MY_HC12_BAUD_RATE (9600)
+#endif
+
+/**
+ * @def MY_HC12_MAX_MESSAGE_LENGTH
+ * @brief The maximum message length used for HC12.
+ */
+#ifndef MY_HC12_MAX_MESSAGE_LENGTH
+#define MY_HC12_MAX_MESSAGE_LENGTH (40)
+#endif
+
+
+/**
+ * @def MY_HC12_SOH_COUNT
+ * @brief Use this in case of collisions on the bus. 3 might be a good setting.
+ */
+#ifndef MY_HC12_SOH_COUNT
+#define MY_HC12_SOH_COUNT (1)
+#endif
+
+/**
+ * @def MY_HC12_SET_PIN
+ * @brief Define this to change the chip enable pin from the default.
+ */
+
+#ifndef MY_HC12_SET_PIN
+#if defined MY_RADIO_HC12
+//#warning SET is not defined, Remember to set before include of mysensor.h if needed
```

Comment:
SET->MY_HC12_SET_PIN, otherwise people might try to define SET which won't do any good.
"mysensor.h" -> "MySensors.h"

Reply:
I runs fine, but cannot be configured, meaning it will run with was it was manual configured with, so if power was set to P1 it will not be changed